axis_backend/apps/clients/repositories/__init__.py

```
@@ -0,0 +1,5 @@
+"""Repository layer for Clients app."""
+from .industry_repository import IndustryRepository
+from .client_repository import ClientRepository
+
+__all__ = ['IndustryRepository', 'ClientRepository']
```

Ruff (0.14.5)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)